test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -78,6 +78,7 @@
     ("aten::slow_conv_transpose2d_backward", datetime.date(2022, 1, 31)),
     ("aten::slow_conv_transpose3d", datetime.date(2022, 1, 31)),
     ("aten::slow_conv_transpose3d_backward", datetime.date(2022, 1, 31)),
+    ("aten::index_copy_", datetime.date(2022, 5, 31)),
```

Comment:
oh, I think this should be `_index_copy_`? https://github.com/pytorch/pytorch/runs/5071946764?check_suite_focus=true

Reply:
Yes, thanks Brian. Have fixed this now, will wait for the tests to pass now :)